ebean-core/src/main/java/io/ebeaninternal/server/deploy/BeanMergeHelp.java

```diff
@@ -172,13 +159,14 @@ public EntityBean mergeBeans(BeanDescriptor<?> desc, EntityBean bean, EntityBean
     if (existing == null && !isNullOrZero(id)) {
       if (desc.isReference(bean._ebean_getIntercept())) {
         existing = (EntityBean) desc.createRef(id, persistenceContext);
-      } else {
+      } else if (hitDb) {
         // Now, we must hit the database and try to find the reference bean
         existing = (EntityBean) desc.findReferenceBean(id, persistenceContext);
```

Comment:
Wir schauen bei Referenzen nur dann in der DB nach, wenn es nicht "orphanremoval" ist.
Bei "orphanremoval" ist es meiner Meinung nach nicht möglich, eine Bean "umzuhängen"
Somit sind wir entweder selber im Besitz der Bean mit der ID (und ist damit im PC), oder die Bean gibt es nicht